concrete-csprng/src/generators/implem/aesni/mod.rs

```diff
@@ -5,7 +5,6 @@
 //! [intel aesni white paper 323641-001 revision 3.0](https://www.intel.com/content/dam/doc/white-paper/advanced-encryption-standard-new-instructions-set-paper.pdf).
 
 mod block_cipher;
-use block_cipher::*;
```

Comment:
Why remove this use?

Reply:
Well, apparently the import is not necessary (clippy is complaining). Don't know why I missed it in the first place though !

Reply:
Ok, got it, I fixed the problem in the upstream PR :wink: